portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -418,6 +418,52 @@ definition {
 		AssertTextNotPresent(value1 = "WC Title");
 	}
 
+	@description = "This is a use case for LPS-110393."
+	@priority = "4"
+	test AssertExpandPanelsOnGlobalScopes {
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "OAuth 2",
+			configurationName = "Bundle PrefixHandlerFactory",
+			configurationScope = "System Scope");
+
+		SystemSettings.gotoConfigurationEntry(configurationEntry = "Default");
+
+		SystemSettings.deleteConfiguration();
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "OAuth 2 Administration");
+
+		LexiconEntry.gotoEntryMenuItem(
+			menuItem = "Edit",
+			rowEntry = "Analytics Cloud");
+
+		AssertClick(
+			key_navItem = "Scopes",
+			locator1 = "NavBar#NAV_ITEM_LINK",
+			value1 = "Scopes");
+
+		AssertElementPresent(
+			key_navItem = "Scopes",
+			locator1 = "NavBar#ACTIVE_NAV_ITEM_LINK");
+
+		Navigator.gotoNavUnderline(navUnderline = "Global scopes");
+
+		Pause(locator1 = "1000");
+
+		for (var globalPanel : list "analytics.read,analytics.write,everything.write,everything.read,everything") {
+			OAuth2.expandGlobalPanel(globalPanel = "${globalPanel}");
+
+			Pause(locator1 = "1000");
```

Comment:
@dellawang Can we change these pauses to be assertions instead? Portal performance may fluctuate meaning that sometimes a longer pause would be needed.

Reply:
Hi @xbrianlee, I remove these pauses since that already be checked in `expandGlobalPanel`.